ReactWindows/ReactNative/Views/Picker/ReactPickerManager.cs

```diff
@@ -0,0 +1,210 @@
+using Windows.UI.Xaml.Media;
+
+namespace ReactNative.Views.Picker
+{
+    /// <summary>
+    /// A view manager responsible for rendering picker.
+    /// </summary>
+    public class ReactPickerManager : BaseViewManager<ComboBox, ReactPickerShadowNode>
+    {
+        private const string PROP_COLOR     = "color";
+        private const string PROP_ENABLED   = "enabled";
+        private const string PROP_ITEMS     = "items";
+        private const string PROP_LABEL     = "label";
+        private const string PROP_SELECTED  = "selected";     
+
+        private int _selected;
+
+        /// <summary>
+        /// The name of the view manager.
+        /// </summary>
+        public override string Name
+        {
+            get
+            {
+                return "RCTPicker";
+            }
+        }
+
+        /// <summary>
+        /// Sets whether a picker is enabled.
+        /// </summary>
+        /// <param name="view">a combobox view.</param>
+        /// <param name="enabled">
+        /// Set to <code>true</code> if the picker should be enabled,
+        /// otherwise, set to <code>false</code>.
+        /// </param>
+        [ReactProperty(PROP_ENABLED)]
+        public void SetEnabled(ComboBox view, bool enabled)
+        {
+            view.IsEnabled = enabled;
+        }
+
+        /// <summary>
+        /// Sets the selected item.
+        /// </summary>
+        /// <param name="view">a combobox instance.</param>
+        /// <param name="selected">The selected item.</param>
+        [ReactProperty(PROP_SELECTED)]
+        public void SetSelected(ComboBox view, int selected)
+        {
+            // Temporarily disable selection changed event handler.
+            view.SelectionChanged -= OnSelectionChanged;
+
+            _selected = selected >= -1 ? selected : -1;
+            view.SelectedIndex = view.Items.Count > _selected ? _selected : view.Items.Count - 1;
+
+            if (view.SelectedIndex != -1 )
+            {
+                view.Foreground = ((ComboBoxItem)(view.Items[view.SelectedIndex])).Foreground;
+            }
+
+            view.SelectionChanged += OnSelectionChanged;
+        }
+
+        /// <summary>
+        /// Populates a <see cref="ComboBox"/>
+        /// </summary>
+        /// <param name="view">a combobox instance.</param>
+        /// <param name="items">The picker items.</param>
+        [ReactProperty(PROP_ITEMS)]
+        public void SetItems(ComboBox view, JArray items)
+        {
+            view.SelectionChanged -= OnSelectionChanged;
+
+            for (int index = 0; index < items.Count; index++)
+            {
+                JToken token;
+                if ( (items[index] as JObject).TryGetValue(PROP_LABEL, out token) )
+                {
+                    var item = new ComboBoxItem();
+                    JToken color;
+                    item.Content = token.Value<string>();
+                    if (((JObject)(items[index])).TryGetValue(PROP_COLOR, out color) )

```

Comment:
You can skip the casting and out parameter by doing something like:

items[index].Value<uint?>(PROP_COLOR);

This will keep your scoping down.  Note that if an object does not contain the key provided to the value method, it returns default(T), which in the case of a nullable is null.